src/Microsoft.PowerShell.Commands.Utility/commands/utility/WriteProgressCmdlet.cs

```diff
@@ -17,10 +17,9 @@ public sealed class WriteProgressCommand : PSCmdlet
         /// </summary>
         [Parameter(
             Position = 0,
-            Mandatory = true,
             HelpMessageBaseName = HelpMessageBaseName,
             HelpMessageResourceId = "ActivityParameterHelpMessage")]
-        public string Activity { get; set; }
+        public string Activity { get; set; } = " ";
```

Comment:
Public ProgressRecord constructor has check IsNullOrEmpty and will throw. Perhaps we need new internal constructor.

Reply:
There's a whitespace there. IsNullOrEmpty allows strings that only contain whitespace so there shouldn't be any issues. `"."` could be a safer option in case there's some `IsNullOrWhiteSpace` checks that I've missed but that might look a little odd in the progress rendering of some hosts.